### PR TITLE
Consistent naming

### PR DESCRIPTION
**Pull request by andreysobol**
- make _calculateEmptySubtreeHash and _updateNode and _modifyTree private (not internal)
- remove _ from every function in PublicSmt.metasol

### CODE REVIEW

File: contracts/Smt.metasol [hunk 1]
```diff
@@ -20,7 +20,7 @@ library SMT{{hashname}} {
         require(_depth < 255, "Depth must be < 255");
         tree.emptySubtreeCache[0] = {{hash}}("");
         // current depth == 0
-        tree._calculateEmptySubtreeHash(_depth);
+        _calculateEmptySubtreeHash(tree, _depth);
```
Comment (ly0va) [inline contracts/Smt.metasol:23]: Why this change? If you're gonna stick with it, should probably remove `using SMT{{hashname}} for MerkleTree` line

Comment (andreysobol) [reply contracts/Smt.metasol:23]: >Why this change?

if it's private method (`_calculateEmptySubtreeHash` now private) i can not call `tree._calculateEmptySubtreeHash(_depth)` i need to use `_calculateEmptySubtreeHash(tree, _depth)`

>If you're gonna stick with it, should probably remove using `SMT{{hashname}} for MerkleTree` line

i don't get it sorry

**Review by ly0va — APPROVED** [review]

**Review by vladbochok — APPROVED** [review]

**Review by barakshani — APPROVED** [review]

**Review by miladpiri — APPROVED** [review]

**Review by Barichek — APPROVED** [review]
